doc/whats_new/v1.0.rst

```diff
@@ -257,6 +257,10 @@ Changelog
   :class:`ensemble.StackingClassifier` and :class:`ensemble.StackingRegressor`.
   :pr:`19564` by `Thomas Fan`_.
 
+- |Fix| Fixed the range of the argument max_samples to be (0.0, 1.0]
+  in :class:`ensemble.BaseForest` and the class that inherits it.
+  :pr:`20159` by `murata-yu`_.
```

Comment:
```suggestion
  :pr:`20159` by :user:`<murata-yu>`.
```

Reply:
thanks, @cmarmo ! I was just in trouble. Fixed it on https://github.com/scikit-learn/scikit-learn/pull/20159/commits/b803bf74eda1f36fefb957234ca75e008b3d90ce.